packages/wrangler/src/create-worker-preview.ts

```diff
@@ -122,13 +130,15 @@ export async function createPreviewSession(
 		? `/zones/${ctx.zone}/workers/edge-preview`
 		: `/accounts/${accountId}/workers/subdomain/edge-preview`;
 
-	const { exchange_url } = await fetchResult<{ exchange_url: string }>(
+	let { exchange_url } = await fetchResult<{ exchange_url: string }>(
 		initUrl,
 		undefined,
 		undefined,
 		abortSignal
 	);
 
+	exchange_url = switchHost(exchange_url, ctx.host).toString();
```

Comment:
Rather than changing this to let and re-assigning the variable, it could stay `const` give the new expression its own variable name expliciting defining what has been done to it `switchedExchangeURL` which can be passed into the `fetch` at `line 144` (only place I saw it being used) 

```suggestion
	const { exchange_url } = await fetchResult<{ exchange_url: string }>(
		initUrl,
		undefined,
		undefined,
		abortSignal
	);

const switchedExchangeURL = switchHost(exchange_url, ctx.host).toString();
```
On line 144:
```ts
const exchangeResponse = await fetch(switchedExchangeURL, { signal: abortSignal });
```

Reply:
Much nicer, yeah!